### PR TITLE
ml-platform release from development branch

### PR DESCRIPTION
**Pull request by arueth**

### CODE REVIEW

File: best-practices/ml-platform/test/README.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+  touch ${HOME}/.kaggle/kaggle.json
+  chmod 600 ${HOME}/.kaggle/kaggle.json
+
+  # Put your API token in the secure file using your preferred editor
+  nano ${HOME}/.kaggle/kaggle.json
+  ```
+
+## End to end tests
+
+### Playground BYOP GitHub Dataprocessing
```
Comment (karajendran) [inline best-practices/ml-platform/test/README.md:66]: add MLP_GIT_NAMESPACE variable.

File: best-practices/ml-platform/test/README.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+- Set GitHub owner
+
+  ```
+  export MLP_GITHUB_ORG=
+  ```
+
+- Set GitHub user
+
+  ```
+  export MLP_GITHUB_USER=
```
Comment (karajendran) [inline best-practices/ml-platform/test/README.md:79]: MLP_GIT_USER_NAME

File: best-practices/ml-platform/test/README.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+  ```
+
+- Set GitHub email
+
+  ```
+  export MLP_GITHUB_EMAIL=
```
Comment (karajendran) [inline best-practices/ml-platform/test/README.md:85]: MLP_GIT_USER_EMAIL

**Review by karajendran — APPROVED** [review]
Ran the test ai-on-gke/best-practices/ml-platform/test/scripts/unit/playground_byop_gh_apply.sh.

_**Apply complete! Resources: 64 added, 0 changed, 0 destroyed.

Outputs:

configsync_repository = "https://github.com/karajendran/mlp-configsync-dev"
git_repository = "github.com/karajendran/mlp-configsync-dev"
iap_domain = "gcp.solutions"
ray_dashboard_url_https = "https://ray-dashboard.ml-team.mlp-dev.endpoints.gkebatchexpce3c8dcb.cloud.goog"
[OK]

Total runtime for terraform_apply: 00:22:41**_

**Comment by karajendran** [comment]
Able to destroy successfully with 'ai-on-gke/best-practices/ml-platform/test/scripts/unit/playground_byop_gh_destroy.sh'

Destroy complete! Resources: 64 destroyed.
[OK]
Total runtime for terraform_destroy: 00:12:08

**Review by alizaidis — APPROVED** [review]
LGTM

**Pull request merged**